sympy/physics/control/lti.py

```diff
@@ -249,6 +261,109 @@ def expand(self):
         """
         return TransferFunction(expand(self.num), expand(self.den), self.var)
 
+    def dc_gain(self):
+        """
+        Computes the gain as frequency approaches zero.
```

Comment:
Nitpick; transfer functions don't have frequencies they have frequency responses. It's a minor thing but if we are going for precision it might matter.

Reply:
Perhaps
```suggestion
        Computes the gain of the response as the frequency approaches zero.
```